netty/src/test/java/io/grpc/netty/NettyServerTest.java

```diff
@@ -239,8 +241,13 @@ public void serverShutdown() {
         shutdownCompleted.set(null);
       }
     });
+
     assertThat(((InetSocketAddress) ns.getListenSocketAddress()).getPort()).isGreaterThan(0);
 
+    Socket socket = new Socket();
+    socket.connect(ns.getListenSocketAddress(), /* timeout= */ 8000);
+    socket.close();
+    countDownLatch.await(); // SocketStats won't be available until binding complete.
```

Comment:
Binding is complete when start() returns, so I don't think this comment is accurate.